docker/tracker_dcs_stack/docker-compose.dummy.yml

```diff
@@ -1,46 +1,45 @@
 version: "3"
 services:
   hv:
-    image: ghcr.io/cdozen/dummy:0.0.1
+    image: ghcr.io/cms-tedd-ip2i/dummy:main 
```

Comment:
use fixed tags, as I told you already, main is a moving target.

Reply:
I can't push tag to the official dummy/low_voltage repo. I don't have access rights.

Reply:
added official repo and fix tag: ghcr.io/cms-tedd-ip2i/dummy:0.0.1